.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:11-stretch-browsers
+    steps:
+      - checkout
+      - run:
+          command: mvn clean verify
+      - when:
+          condition:
+            equal: [master, << pipeline.git.branch >>]
+          steps:
+            - run:
+                name: Sonar Analysis
```

Comment:
any specific reason you did not use the orb?

Reply:
the orb is using the sonar-scanner. For maven project, it is better to use the sonar-maven-plugin (which is already configured in our projects)

Reply:
Thanks for explaining.